.github/workflows/publish-hapi-api-protos.yml

```
@@ -0,0 +1,46 @@
+on:
+  release:
+    types: [published]
+
+name: hapi-proto-publish
+jobs:
+  check-protos-released:
+    runs-on: [ARM64, self-hosted, Linux]
+    steps:
+      - name: Check Release
+        id: hapi_proto_released
+        uses: actions/github-script@v5
+        with:
+          result-encoding: string
+          script: return context.ref.includes('refs/tags/hapi-proto-v')
+    outputs:
+      hapi_proto_released: ${{ steps.api_released.outputs.result }}
+  npm-publish:
+    runs-on:  [ARM64, self-hosted, Linux]
+    if: ${{ needs.check-protos-released.outputs.hapi_proto_released == 'true' }}
```

I think you need a `needs` field here?
```suggestion
    if: ${{ needs.check-protos-released.outputs.hapi_proto_released == 'true' }}
    needs:
      - check-protos-released
```